cset-workflow/meta/rose-meta.conf

```diff
@@ -218,7 +218,9 @@ compulsory=true
 ns=Data and Cycling
 description=Datetime of initial cycle point.
 help=This should be an ISO 8601 datetime string indicating when you want the
-    cycling component of CSET to begin. E.g. 2000-01-01T00:00Z
+    cycling component of CSET to begin. E.g. 2000-01-01T00:00Z. Ensure that it is
+    consistent with your data starting time. If not then a warning that cubes
```

Comment:
```suggestion
    consistent with your data starting validity time. If not then a warning that cubes
```

Reply:
Suggesting adding validity for extra clarity that it is to cycle over validity times rather than forecast cycle.